src/components/v2/V2Project/NftRewardsSection/NftPreview.tsx

```diff
@@ -0,0 +1,118 @@
+import { Trans } from '@lingui/macro'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+import { NftRewardTier } from 'models/v2/nftRewardTier'
+import { CSSProperties, useContext } from 'react'
+import { LinkOutlined, CloseOutlined } from '@ant-design/icons'
+import ExternalLink from 'components/ExternalLink'
+import { DEFAULT_NFT_MAX_SUPPLY } from 'hooks/v2/NftRewards'
+
+import { darkColors } from 'constants/styles/colors'
+
+const containerStyle: CSSProperties = {
+  zIndex: 10000,
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  width: '100%',
+  height: '100%',
+  backgroundColor: 'rgba(0,0,0,0.8)',
+  overflow: 'auto',
+}
+
+const headerRowStyle: CSSProperties = {
+  display: 'flex',
+  width: '100%',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  marginBottom: '15px',
+}
+
+const headerStyle: CSSProperties = {
+  marginBottom: 0,
+  color: darkColors.light0,
+  fontWeight: 500,
+}
+
+const linkStyle: CSSProperties = {
+  marginLeft: '25px',
+  display: 'flex',
+  alignItems: 'center',
+  cursor: 'pointer',
+  color: darkColors.light0,
+}
+
+export function NftPreview({
```

Comment:
Decided it would be easier to just our own whole preview rather than leverage Antd. A lot of customization and would have been a fuck around